orchestrator/app.py

```diff
@@ -180,6 +187,26 @@ def register_subscription_models(product_to_subscription_model_mapping: Dict[str
         """
         SUBSCRIPTION_MODEL_REGISTRY.update(product_to_subscription_model_mapping)
 
+    def register_graphql(self: "OrchestratorCore", query: Any = Query, mutation: Any = Mutation) -> None:
```

Comment:
The graphql endpoint used to be enabled by default but now it looks like it's opt-in, as this function is not called in `OrchestratorCore.__init__`. Is that intentional?

Reply:
yes, because the domain models can only be auto registered after the products are imported, otherwise the `REGISTERED_DOMAIN_MODELS` will be empty.

Reply:
Ah right, that makes sense.

Makes me think we should come up with a better way for users to register domain models.. something that would automatically include the graphql model generation

Reply:
I believe that this is the best way currently, to only do it when registering the graphql model. You can't register anything after `register_graphql`, because it won't be recognized anymore once the graphql is initialized.